design-document.md

```diff
@@ -14,4 +24,169 @@
 
 ## Open questions and system parameters
 
-## Cryptographic dependencies
+## Cryptographic components
+
+### Overview of the core protocol
+
+**Notation.**
+Fix a finite field K. We write the linear, s-way secret sharing of an element x
```

Comment:
It's worth adding in this flexibility now in case we end up needing it later. Depending on our security considerations and how address resilience, it may end up being useful to have 3 or more aggregators.